locales/en/rle-verified.json

```diff
@@ -0,0 +1,13 @@
+{
+    "rle_verified_title": "RLE confirmed",
```

Comment:
In the prototype v5, the title is 'PSC confirmed'. I guess we better match this one :).